### PR TITLE
prometheus: enable OAuth Grant flow for configured users

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
#0

### CODE REVIEW

File: cluster/manifests/prometheus/ingress.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# TODO: Remove
```
Comment (szuecs) [inline cluster/manifests/prometheus/ingress.yaml:1]: ^

Comment (AlexanderYastrebov) [reply cluster/manifests/prometheus/ingress.yaml:1]: I want to deploy RouteGroup and remove Ingress in the followup PR such that there is no down time (couple of seconds possible between deletion and Skipper discovering route group).

Comment (szuecs) [reply cluster/manifests/prometheus/ingress.yaml:1]: There is nothing that depend on it that can not deal with a downtime, but if you want to make it 0-downtime it's also fine.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**